fix(lib): change tag min-width (#3516)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/src/content/docs/foundation/colors.mdx

## Decorative

<Callout type="info" title="For design only">

>   Decorative colors are used for illustrations design purpose only, they should not be needed in OUDS Web. Please contact us if you need to use it in your project.

Reviewer: Decorative colors are used for coloring illustrations only and should not be needed in OUDS Web. Please contact us if you need to use them in your project.

>   Decorative colors are used for illustrations design purpose only, they should not be needed in OUDS Web. Please contact us if you need to use it in your project.

Reviewer 2: ```suggestion
  Decorative colors are used for coloring illustrations only and should not be needed in OUDS Web. Please contact us if you need to use them in your project.
```
